zebrad/src/commands/start.rs

```diff
@@ -55,13 +55,48 @@ impl StartCmd {
         ));
 
         info!("initializing verifiers");
-        let verifier = zebra_consensus::chain::init(
+        let chain_verifier = zebra_consensus::chain::init(
             config.consensus.clone(),
             config.network.network,
             state.clone(),
         )
         .await;
 
+        let block_verifier =
+            zebra_consensus::block::BlockVerifier::new(config.network.network, state.clone());
+
+        // 1- get the finalized tip
+        let tip = match state.clone().oneshot(zebra_state::Request::Tip).await {
+            Ok(zebra_state::Response::Tip(Some(tip))) => tip,
+            Ok(zebra_state::Response::Tip(None)) => (
+                zebra_chain::block::Height(1),
+                zebra_chain::parameters::genesis_hash(config.network.network),
+            ),
+            _ => unreachable!("wrong response to Request::Tip"),
+        };
+
+        // 2- skip check if the block height is below the mandatory checkpoint
+        let tip_height = tip.0;
+        if tip_height >= config.network.network.mandatory_checkpoint_height() {
```

Comment:
> skip the finalized tip check if its height is less than or equal to the mandatory checkpoint

See #2308

If you skip the final checkpoint block, you can use the chain verifier here, because the block will automatically go to the block verifier:
```suggestion
        if tip_height > config.network.network.mandatory_checkpoint_height() {
```